downstream/modules/platform/proc-aap-hub-restore.adoc

```diff
@@ -0,0 +1,30 @@
+[id="aap-hub-restore"]
+
+= Recovering the {HubNameStart} deployment
+
+[role=_abstract]
+Use this procedure to restore a previous hub deployment into the namespace. The deployment name you provide will be the name of the new AutomationHub custom resource that will be created.
+
+[NOTE]
+====
+The name specified for the new AutomationHub custom resource must not match an existing deployment or the recovery process will fail.
```

Comment:
I notice we have this noted twice in this section...  But I'm not wholly against it.  This is an important point I want to make sure customers see.